test/sqllogictest/joins.slt

```diff
@@ -169,7 +169,7 @@ WHERE l1.la IN (
 
 %4 =
 | Get %2 (l0)
-| Filter (#0 = (#1 + 1))
+| Filter !(isnull(#0)), ((#0 = (#1 + 1)) || (isnull(#0) && isnull(#1)))
```

Comment:
Also interesting!

Reply:
Indeed, this is interesting. (Though I consider resolving removing the `(isnull(#0) && isnull(#1)` in light of the presence of `!(isnull(#0))` to be for another PR another day.) 

Disturbingly, there was a point in this PR where I did not add the `|| (isnull(...) && isnull(...)` when pushing down single-input join equivalences, and it passed all of SLT. It was only a correctness failure in DBT that clued me to this oversight. I'm planning on adding a regression test to SLT for this.